db/migrate/20220928010541_initialize_people_table.rb

```diff
@@ -0,0 +1,18 @@
+class InitializePeopleTable < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
This is in preparation for user profiles (Lesson 19) which I haven't completed yet